lib/Doctrine/DBAL/Schema/PostgreSqlSchemaManager.php

```diff
@@ -65,7 +66,7 @@ public function getSchemaNames()
     {
         $rows = $this->_conn->fetchAll("SELECT nspname as schema_name FROM pg_namespace WHERE nspname !~ '^pg_.*' and nspname != 'information_schema'");
 
-        return array_map(function ($v) { return $v['schema_name']; }, $rows);
+        return array_column($rows, 'schema_name');
```

Comment:
Probably not worth reworking but `fetchAll(FetchMode::COLUMN)` would be a better approach (also would eliminate the need for the column alias).

Reply:
Let me check first: if it not many efforts, I'll do the changes (like the proposal ;))